utils/config_goerli.py

```diff
@@ -11,21 +11,42 @@
 # Our custom Aragon apps
 lido_dao_node_operators_registry = "0x9D4AF1Ee19Dad8857db3a45B0374c81c8A1C6320"
 lido_dao_steth_address = "0x1643E812aE58766192Cf7D2Cf9567dF2C37e9B7F"
-lido_dao_oracle = "0x24d8451BC07e7aF4Ba94F69aCDD9ad3c6579D9FB"
+lido_dao_legacy_oracle = "0x24d8451BC07e7aF4Ba94F69aCDD9ad3c6579D9FB"
 # Aragon APM Repos
 lido_dao_voting_repo = "0x14de4f901cE0B81F4EfcA594ad7b70935C276806"
 lido_dao_lido_repo = "0xe9ede497d2417fd980d8b5338232666641b9b9ac"
 lido_dao_node_operators_registry_repo = "0x5f867429616b380f1ca7a7283ff18c53a0033073"
-lido_dao_oracle_repo = "0x9234e37Adeb44022A078557D9943b72AB89bF36a"
+lido_dao_legacy_oracle_repo = "0x9234e37Adeb44022A078557D9943b72AB89bF36a"
 
 curve_rewards_manager_address = ""
 balancer_rewards_manager = ""
 
-lido_dao_deposit_security_module_address = "0x7DC1C1ff64078f73C98338e2f17D1996ffBb2eDe"
+lido_dao_deposit_security_module_address_old = "0x7DC1C1ff64078f73C98338e2f17D1996ffBb2eDe"
+lido_dao_deposit_security_module_address = "0x07c1e7eDC8a19c9A9b936DecDd531a6365D2a434"
 
-lido_dao_composite_post_rebase_beacon_receiver = "0x5d2113f7691ac6Df5E3f41Fb938429ACEAD2C94f"
-lido_dao_self_owned_steth_burner = "0x6432756feF0fb527C06eFd4689A7CE0E195bD327"
-lido_dao_execution_layer_rewards_vault = "0x94750381bE1AbA0504C666ee1DB118F68f0780D4"
+# Needed only for the tests development. Before the voting start the LidoLocator
+# proxy is to be set to the implementation
+lido_dao_lido_locator_implementation = "0xf47F64a3B2AA52A77dC080D50927Af378d7dA7B8"
+
+# Needed temporary only for Shapella upgrade
+deployer_eoa = "0xa5F1d7D49F581136Cf6e58B32cBE9a2039C48bA1"
+
+lido_dao_lido_locator = "0x1eDf09b5023DC86737b59dE68a8130De878984f5"
+lido_dao_burner = "0x20c61C07C2E2FAb04BF5b4E12ce45a459a18f3B1"
+lido_dao_execution_layer_rewards_vault = "0x27Dec30CC9EfDDDAF04C7CA2737759CBf6405ecb"
+lido_dao_hash_consensus_for_accounting_oracle = "0x821688406B8000FE3bAa8B074F8e1CbCD72c0035"
+lido_dao_accounting_oracle = "0x5864490e56B6f013Bc702Cf137Ed32Dc2C86F818"
+lido_dao_hash_consensus_for_validators_exit_bus_oracle = "0xe47EA5f0406C1A976cE43f97cEdcB8f3dee5484A"
+lido_dao_validators_exit_bus_oracle = "0xb75A55EFab5A8f5224Ae93B34B25741EDd3da98b"
+lido_dao_oracle_report_sanity_checker = "0x43c7FA62b3FD800d02edb1d38fd5b079d3ff84c6"
+lido_dao_withdrawal_queue = "0xCF117961421cA9e546cD7f50bC73abCdB3039533"
+gate_seal = "0x75A77AE52d88999D0b12C6e5fABB1C1ef7E92638"
+
+
+lido_dao_eip712_steth = "0xB4300103FfD326f77FfB3CA54248099Fb29C3b9e"
+lido_dao_withdrawal_vault = "0x70760635A8862c562C5Ad0374D7f9CEA4A7193D2"
```

Comment:
our creds on Görli are:
0x010000000000000000000000dc62f9e8c34be08501cdef4ebde0a280f576d762

So the contract should be https://goerli.etherscan.io/address/0xdc62f9e8c34be08501cdef4ebde0a280f576d762#readContract

Reply:
fixed, it's a suuuper remark